### ISSUE TITLE
Sound Damage: Sonic or Thunder?

### ISSUE THREAD

**Issue by DalenWBrauner** [mlenser/kryx-rpg-issues#327]
Kryx & I discussed briefly on Discord whether Thunder damage belongs primarily in the Sound theme or the Storm theme. I see the damage type as being purely sound-based, and thought "Sonic damage" might be a more accurate term. He suggested we hash this out first before addressing the other Sound/Storm related issues (e.g. #320,#321, #322, #325)

The english language argument is that "Thunder" is a _type_ of Sound, whereas "Sonic" is a word to describe something being "of sound" or "relating to sound".

[This Stack Exchange post](https://rpg.stackexchange.com/questions/92990/is-there-a-difference-between-thunder-force-and-bludgeoning-damage), I feel, helps illustrate exactly what Thunder damage means in 5e:
> Caused by a sonic boom, or otherwise extremely loud sound, representing the concussive force and damaging effect on the ear.
> A concussive burst of sound, such as the effect of the thunderwave spell, deals thunder damage.

Kryx pointed out that all previous editions of D&D used to call it Sonic damage, but aside from [an unhelpful reddit thread](https://www.reddit.com/r/DnD/comments/321ffc/thunder_damage_5e/?utm_content=comments_view_all) I couldn't find any speculation as to why.

Kryx also observed that PF2e currently calls it Sonic damage, though this could be because of its origins in D&D 3.5. Again, I couldn't find a lot of information on _why_ it was called Sonic over Thunder, perhaps because it wasn't considered. At a glance, Pathfinder has some interesting examples of sound-related spells, so we could pull more inspiration from there.

My vote would be to rename Thunder damage to Sonic damage, but (as always) would appreciate additional opinions, research and insight.

**Comment by itamarcu** [comment]
I think this quote gives a pretty good reason:
> "Sonic" didn't fit with the fantasy theme, and "Thunder" sounds better. Electricity is "Lightning", sound is "Thunder".

Lightning is just one of many effects that can electrocute someone, but the system names the damage type after it, because the people of this fantasy world know what lightning does even if they don't know what an electric fence does. It sounds more medieval and less high-tech, and Sonic is much the same way. It adds to the fantasy feeling, even if it's much less accurate.

Personally, I think Thunder damage feels very similar to Force damage and in a way Bludgeoning damage. There's differences, but the differences are mainly about the source of the damage, and it's hard to really understand in your imagination how thunder damage can hurt a creature with no ears, and how it can do it in a different way than force or bludgeoning does. They're all just a concussive kinetic force, aren't they?

**Comment by DalenWBrauner** [comment]
>Personally, I think Thunder damage feels very similar to Force damage and in a way Bludgeoning damage.

This topic is covered thoroughly in [that Stack Exchange post](https://rpg.stackexchange.com/questions/92990/is-there-a-difference-between-thunder-force-and-bludgeoning-damage), and I'd rather stick with that. It explicitly states:
> Caused by a sonic boom, or otherwise extremely loud sound, representing the concussive force and damaging effect on the ear.

I'd rather keep with this flavor; sound-based damage that takes the form of concussive forces, piercing screeches, and other high-decible sounds. There are plenty of characters in media who can break glass, inflict pain, or blast away characters & objects with sound; otherwise they wouldn't exist in the other systems (D&D/PF).

You could argue this is a bit "encompassing", but I could argue 'so are the other elemental damage types'. We don't _have_ to split hairs on "eardrum damage vs concussive forces" in the same way we don't _have_ to split hairs on "hot metal vs on fire vs hot ash vs exploding meteors" for Fire damage. It's all part of the same understood thematic concept.
 
>  it's hard to really understand in your imagination how thunder damage can hurt a creature with no ears

Plenty of things in D&D do not explicitly say they "are immune to psychic damage" despite not really having a mind. Similarly, we don't need to solve for every case as long as we have enough guidance on common examples to extrapolte from. Much of the game is improvising and split-DM-decisions based on what they think makes sense (+ existing rules).
We already have a simple rule for this:
> A deafened creature is immune to auditory effects.

From this, I think we can extrapolate that creatures that can hear- yes, even skeletons, headless zombies, magical constructs and other earless creatures- can take Thunder damage as long as they are not Deafened. I also think we can assume that your ability to hear has nothing to do with whether you would be physically pushed back with the concussive force.

These are what I'd call "solved problems"- I believe the writers of D&D and Pathfinder already have a functioning conclusion for most of it. (But normally I appreciate that "let's break it down" approach!)

>"Sonic" didn't fit with the fantasy theme, and "Thunder" sounds better.

I don't think "Thunder" is quite what I'd use to describe the damage taken from standing next to the crashing of an oversized pair of symbols or a poorly-played violin, though. The word "Sonic" might not be a _perfect_ fit for a fantasy setting, but I haven't really found a sound-based word that _is_, other than just... "Sound damage" (which is less modern, but less cool).

**Comment by DalenWBrauner** [comment]
That said, I don't actually have a strong opinion on the name. I just want this damage type to 'belong' primarily to the Sound theme, not Storm, as you may be able to guess from my aforementioned issues.

I believe Kryx wants the name to match the flavor however, so I'm willing to work towards that design tenet to help make this happen.

**Comment by Marcloure** [comment]
I think the reason they called it Thunder damage instead of Sonic damage is the same why they called Lightning damage instead of Electric damage, or Fire damage instead of Heat damage. It's less scientific, more fantasy.

**Comment by Lamorak11** [comment]
I think that a bit of overlap between Storm and Sound regarding which gets thunder damage is fine, a relatively small portion of storm spells actually do thunder damage as it stands anyway.

With that said, I think that thunder damage should therefore belong primarily in the sound theme. 

With this in mind, I think it makes more sense to rename thunder damage to sonic damage.

Additional reasoning:
Sometimes, magic from the sound theme may deal sound-based damage but not necessarily be "thunderous". I can imagine an extremely high pitched sound that could deal damage, and it would be much more fitting to call that damage _sonic_ rather than _thunder_.

In conclusion, I think that thunder damage is always sonic, but sonic damage is not always thunderous. Therefore we should use sonic.

It may not be as evocative, but it's both broader and more accurate in its application.

**Comment by itamarcu** [comment]
That's a good argument. A high-pitched sound is definitely not thunderous but it's definitely sonic.

By the way, how about literally calling it "sound damage"?

**Comment by DalenWBrauner** [comment]
That might be best. The word "Sonic" didn't enter common lexicon until the
concept of a plane performing sonicbooms was more well-known (at least
according to my brief research).

**Comment by mlenser** [comment]
I don't think we should use "sound damage". Thunder is a much better name that would be common in D&D times. But if we're expanding to more than just deep noises like high pitched noises than it becomes difficult to name.

Agreed that Sonic isn't right.
![image](https://user-images.githubusercontent.com/235857/84170022-91acb100-aa79-11ea-9847-9bf56f72379e.png)

But not sure what other options there are or if we should even go down this road. Do high pitched noises really cause damage? I think options like Shatter are more like concussion damage.

**Comment by DalenWBrauner** [comment]
> Do high pitched noises really cause damage? I think options like Shatter are more like concussion damage.

Sound-based damage has a fantastical basis in D&D and other media, as discussed above, so that really isn't the issue. Rather than exclude a known fantasy concept on the basis of a lackluster name, I'd rather pick a placeholder and come back to it in the future. 

As I wrote before, I don't see name as being especially critical, as long as the _concept_ (described above) remains the same. The sufficienctly-fantastical name is really the only piece that's missing.

- **Sound damage** sounds lame, but is accurate and works in a D&D setting.
- **Sonic damage** feels out of place, but is cool and accurate.
- **Thunder damage** isn't accurate, but is cool and fits in D&D.

I'll start asking around my table and see if anyone has any better ideas, but I'd really rather this not get in the way of building out the Sound theme.

**Comment by Marcloure** [comment]
Sound damage is actually more in line with other types of damage like fire damage (not fiery damage), acid damage (not acidic damage), lightning damage (not... "lightning-c" damage), poison damage (not poisonous damage). At the same time, we have radiant damage (not radiance damage), and necrotic damage (not necrose damage).

Anyway, I don't really see reason to change. Not all fire damage is caused by fire, any heat related damage is fire damage (heat metal, hot environments, etc).

**Comment by mlenser** [comment]
> Rather than exclude a known fantasy concept

When did I ever say I was excluding it?

> I'd rather pick a placeholder and come back to it in the future.

The name is super important. For the same reason that the Survival theme is problematic: it means different things to different people.

_____________________

I'm really unsure what we're defining.
- Shatter? Why is Shatter as concussion damage not a fit? That's basically what thunder is.
- Harpy Screams? That's Psychic damage.

I am unsure if "sound damage" is even a thing. High pitched noises don't cause damage.


> I'd really rather this not get in the way of building out the Sound theme.

It is essential to understand what we're building. We can't build a building without a plan. We're still waiting on Survival to be defined so I'd rather do it properly this time.

**Comment by Lamorak11** [comment]
> High pitched noises don't cause damage.

They do in science fiction and fantasy. 

https://smallville.fandom.com/wiki/Sonic_Scream

https://tvtropes.org/pmwiki/pmwiki.php/Main/MakeMeWannaShout?from=Main.SonicScream

**Comment by mlenser** [comment]
> Sonic screams are the sound/sonic-based ability to generate incredibly strong hypersonic sound blasts
> The power of sound. Able to shatter glass, crumble rocks, and knock down buildings. In the world of fiction, anyways.

Sounds like concussion damage, or as its called in D&D: thunder damage.

**Comment by mlenser** [comment]
My view, currently: The damage we're talking about is concussion damage or shock waves. Blast waves that hurt you the same as a bomb or grenade (not the shrapnel kind) would do. That's exactly what Shatter and Thunderwave are. Fire Bomb for example does half fire and half bludgeoning, but it should do half thunder, not half bludgeoning. Telekinesis might also do thunder, not sure yet.

**Comment by Lamorak11** [comment]
Your quotes from those links are somewhat incomplete there, emphasis mine:

> Sonic screams are the sound/sonic-based ability to generate incredibly strong hypersonic sound blasts of a different and **higher**, louder amplitude or **frequency**

And continuing from your TV tropes quote:

> Characters who have super-powerful screams or other sound-based superpowers can usually do something like vocal mimicry, scream out sonic booms, and **screech so loud it can paralyze people by making their eardrums bleed.**

**Comment by mlenser** [comment]
> screech so loud it can paralyze people by making their eardrums bleed.

I would argue everything until the eardrums bleeding is psychic damage. Sound doesn't cause paralysis, imo.

___________________

I'll bold the parts that I believe are important:

> Sonic screams are the sound/sonic-based ability to generate incredibly strong ***hypersonic sound blasts*** of a different and higher, louder amplitude or frequency

Blast waves or concussion damage.

- Psychic = brain damage
- Thunder = concussive damage, often caused by things like shock waves like sonic booms

**Comment by Lamorak11** [comment]
Ok, you've convinced me that very loud high-pitched sounds shouldn't do damage. My very own piercing screech spell doesn't, after all. With you there.

With that in mind, I think it is fine to keep the name as thunder damage. It is evocative, and it really doesn't take a leap of logic to understand that a grenade or a sound-based spell could do thunder damage.

**Comment by mlenser** [comment]
I'm unsure about dividing storm as so many concepts are built around both lightning and thunder..

- Tempest domain: "Gods whose portfolios include the Tempest—including Gozreh, Rovagug, Talos, Umberlee, Kord, Zeboim, the Devourer, Zeus, and Thor—govern storms, sea, and sky. They include gods of lightning and thunder, gods of earthquakes, some fire gods, and certain gods of violence, physical strength, and courage. In some pantheons, a god of this domain rules over other deities and is known for swift justice delivered by thunderbolts."
- Storm Occultist/Suffused subclass: "Your magic comes from the power of elemental air and powerful storms which are turbulent, violent, and destructive. Many with this power can trace their magic back to a near-death experience caused by the Great Rain, but perhaps you were born during a thunderstorm so powerful that folk still tell stories of it, or your lineage might include the influence of potent storm creatures such as storm elementals. Whatever the case, the magic of the storm permeates your being."

Lightning and Thunder seem to go together.

If that were not the case then I'd suggest we create a lightning theme. But because that is the case I suggest all thunder spells return to the Storm theme. Thunderclap, Booming Weapon, Thunderwave, and Shatter.

We are then left with:
- Echolocation feat and spell. I think this should be in the Beast theme
- Manipulate Sound and Control Sound. Control Sound is in Deceit. Manipulate Sound can be added there
- Piercing Screech ?
- Silence ?
- Whispering Wind is in air, it can go there

I think the Sound theme is dead.

**Comment by mlenser** [comment]
Sound theme removed
- Thunderclap, Booming Weapon, Thunderwave, and Shatter remain in the Storm theme
- Echolocation moves to the Beast theme
- Manipulate Sound and Silence move to the Deceit theme and Control Sound remains there
- Piercing Screech moved to the Fear theme
- Whispering Wind remains in the Air theme
